README.adoc

```diff
@@ -49,6 +49,29 @@ To run on an another port use `--server.port=...`:
 java -jar employee-rostering-distribution/target/employee-rostering-distribution-*.jar --server.port=18080
 ----
 
+== Benchmark Application
+
+After building the project above, run the benchmark application with:
+
+[source,shell]
+----
+java -jar employee-rostering-benchmark/target/employee-rostering-benchmark-*.jar
+----
+
+Note: make sure to run the project application and benchmark application on different ports if they are run
+simultaneously.
```

Comment:
How about including the config to change the port in the example above?

Reply:
There are instructions to run a jar on a different port above. Adding the config command again seems redundant.